src/js/themes/base.d.ts

```diff
@@ -672,6 +675,7 @@ export interface ThemeType {
   };
   meter?: {
     color?: ColorType,
+    colors?: string[] | {dark?: string[], light?: string[]},
```

Comment:
Since this line is a duplication of line #190, I'm thinking it should be extracted to it's on type.

Reply:
agreed, done